themes/default/content/blog/observable-infrastructure-as-code/index.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Observable Infrastructure as Code"
+date: 2022-01-20T08:36:11-06:00
+draft: false
+canonical_url: https://dev.to/fllstck/observable-infrastructure-as-code-52ha
+meta_desc: A review of using Honeycomb to add observability to your Pulumi projects
+meta_image: meta.png
+authors:
+    - kay-ploesser
+tags:
+    - community
+    - javascript
+    - observability
+    - automation-api
+---
+
+Pulumi community member [Kay Plößer](https://kay.is/) spent some time digging into setting up observability of a Pulumi deployment using [Honeycomb](https://www.honeycomb.io/). Read more to find out all the details on configuring Honeycomb and Pulumi together, with a side dish of Automation API!
+
+<!--more-->
+Last few weeks, I procrastinated doing real product-related things with setting up monitoring for my infrastructure deployment.
+
+I'm using [@pulumidev](https://dev.to/pulumidev), a cloud-provider independent IaC framework that allows me to define my infrastructure in TypeScript and use whatever provider I need.
+
+I read something about [Honeycomb's](https://www.honeycomb.io/) observability approach to monitoring, and it got me thinking about my deployment code. Later, it will probably run with something like GitHub Actions, so it's not on my machine when it executes. This means I need some insights in case things go wrong. But it would also be good to see how the deployment behaves over time. How do new resources affect deployment durations etc.?
+
+This is what a trace could look like in Honeycomb:
+
+[![Example Trace](./example-trace.png)](./example-trace.png)
+
+Why not setting things up before I move deployment from my development machine to the cloud?
+
+This article will show you how to make your Pulumi deployment observable with [@honeycomb_io](https://dev.to/honeycomb_io)
+
+[](#prerequisites)Pre-Requisites
+---------------------------------------------------------------------------------------------
```

Comment:
Our Hugo system adds the anchor links already, so this actually breaks the links (e.g., this one shows up as http://pulumi-hugo-origin-pr-987-40af68f2.s3-website.us-west-2.amazonaws.com/blog/observable-infrastructure-as-code/#prerequisitespre-requisites).
```suggestion
## Prerequisites
```

Do you want me to update all of them?

Reply:
I'll fix the others after committing this one!